WordPress/Classes/Models/Blog.h

```diff
@@ -31,6 +31,13 @@ typedef NS_ENUM(NSUInteger, BlogFeature) {
     BlogFeaturePushNotifications,
 };
 
+typedef NS_ENUM(NSInteger, SiteVisibility) {
+    SiteVisibilityPrivate = -1,
+    SiteVisibilityHidden = 0,
+    SiteVisibilityPublic = 1,
+    SiteVisibilityUnknown = NSIntegerMax
```

Comment:
NSIntegerMax *might* be different across platforms (32 vs 64 bit) and persisting this value could cause a glitch if the data is restored onto a newer, non-32bit device. The risk and effect is minimal, however.